FEATURE: Player logic

jmysliv opened the pull request:
* Entries are played after being pressed
* Play/Pause/Skipping/Shuffle/Looping works
* Sliding works

TODO:
- [x] visual fixes
- [x] error handling

PR is ready for testing.

alejomendoza commented:
Amazing, works like a charm. Two things I would love to have if possible, a loading state (spinner) and make the slider thumb smaller, see Spotify as a reference.

co012 reviewed (commented):
LGTM

jmysliv commented:
@alejomendoza I've added a loading state and fixed slider bar width jump. Regarding thumb size, the package we are using doesn't allow for such customization. There is an open issue in their repository: https://github.com/callstack/react-native-slider/issues/169
